src/tools/terry_statement.rs

```diff
@@ -0,0 +1,111 @@
+use std::fs;
+use std::path::PathBuf;
+
+use anyhow::{anyhow, Context, Error};
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+
+use crate::LoggerOpt;
+use task_maker_format::terry::TerryTask;
+use task_maker_format::{find_task, EvaluationConfig};
+
+#[derive(Parser, Debug, Clone)]
+pub struct TerryStatementOpt {
+    /// Path to statement template (uses task directory structure if omitted)
+    #[clap(long = "statement-path", short = 's')]
+    pub statement_path: Option<String>,
+
+    /// Path to subtasks files (none if omitted)
+    #[clap(long = "subtasks-path", short = 't')]
+    pub subtasks_path: Option<String>,
+
+    /// Path to store output statement (stdout if omitted)
+    #[clap(long = "output-path", short = 'o')]
+    pub output_path: Option<String>,
+
+    /// Look at most for this number of parents for searching the task
+    #[clap(long = "max-depth", default_value = "3")]
+    pub max_depth: u32,
+}
+
+/// A subtask has a maximum score and a list of testcases
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct StatementSubtask {
+    /// The maximum score on the subtask
+    pub max_score: f64,
+    /// The additional constraints, as seen by the contestant
```

Comment:
Long term, this doesn't quite work well with translations, I think - of course we don't need to care about that now.